sample.js

```diff
@@ -2,15 +2,20 @@ import { ShipthisAPI } from './dist/shipthisapi-js/main.js';
 
 let param = {
   userType: 'employee',
-  organisationId: 'demo',
+  organisationId: 'base-dev',
   xApiKey:
-    '<Your API Key>',
+    'WyJzdWJocmFkd2lwQHNoaXB0aGlzLmNvIiwiYmFzZS1kZXYiXQ.Zsx1xw.y0Kt9MOLi58LX2vMG3RPsSNGDl8',
```

Comment:
don't include xapi key

Reply:
Sorry, I did not noticed that. Now it is fixed